seatunnel-e2e/seatunnel-connector-v2-e2e/connector-paimon-e2e/src/test/java/org/apache/seatunnel/e2e/connector/paimon/CancelStreamJob.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.apache.seatunnel.e2e.connector.paimon;
+
+import org.apache.seatunnel.e2e.common.container.ContainerExtendedFactory;
+
+import org.testcontainers.containers.GenericContainer;
+
+import java.io.IOException;
+
+public class CancelStreamJob implements ContainerExtendedFactory {
+
+    private String jobName;
+
+    public CancelStreamJob(String jobName) {
+        this.jobName = jobName;
+    }
+
+    @Override
+    public void extend(GenericContainer<?> container) throws IOException, InterruptedException {
```

Comment:
why not create new method named `cancelJob` in https://github.com/apache/seatunnel/blob/934434cc75afc747339182df6468f89a5d0fb1b2/seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/TestContainer.java#L28 ?

Reply:
get